docs/studios/unity/unity-sdk/RGAction.mdx

```diff
@@ -8,6 +8,9 @@ Represents an action that a bot can instruct the game object to take.
 This `RGAction` class should be inherited in a sub class, and developers
 will need to implement the `StartAction` and `GetActionName` function.
 
+Also see [`RGFindUtils`](./RGFindUtils) and [`RGState`](./RGState) for concepts that come up when
```

Comment:
If we're telling people to reference another page to "find out how to do x", then it'll need to be more specific. I really like the way you worded it in the first_tutorial.md 

> See the [`RGFindUtils`](../unity-sdk/RGFindUtils) class for helper functions which find GameObjects and components within your scene based on information coming in from an action.

Reply:
Good suggestion! Let me know what you think of my edit.

Reply:
I'd remove the bit about "an easier way" because it's not really a separate method of defining actions, just helpers to do so. It'd also be weird if we were presenting them with "the hard way" first.

Suggest rewording to something like "In most RGAction implementations, it is necessary to find game objects within the scene based on the game state. See the RGState class for defining the game state and the RGFindUtils class for helper functions to locate GameObjects within your scene."

Reply:
Added!